src/erlastic_search.erl

```diff
@@ -648,6 +650,30 @@ aliases(Body) ->
 aliases(Params, Body) ->
     erls_resource:post(Params, filename:join([<<"_aliases">>]), [], [], erls_json:encode(Body), Params#erls_params.http_client_options).
 
+%% Bulk operations with default params
```

Comment:
Please follow the established format for documentation (in particular with a link to the official doc), e.g. https://github.com/AnushaMorappanavar/erlastic_search/blob/41af00ee49837824c47d08fb80f97ccaaff44e1f/src/erlastic_search.erl#L136-L142